### PR TITLE
Simplify override description

### PR DESCRIPTION
**Pull request by TheYoBots**

### CODE REVIEW

**Comment by TheYoBots** [comment]
https://github.com/TheYoBots/lichess-bot/blob/simplify-overrides/wiki/Configure-lichess-bot.md#challenging-other-bots

**Review by MarkZH — APPROVED** [review]

**Comment by TheYoBots** [comment]
Before this is merged, I think the wiki `.md` file can also be simplified further. It still is slightly confusing.

**Comment by TheYoBots** [comment]
It's slightly improved now. Do feel free to check if there is anything else that I need to change.

File: config.yml.default [hunk 1]
```diff
@@ -226,3 +226,5 @@ matchmaking:
 #       - 2
 #       - 3
 #     challenge_mode: casual
+#
+# The following configurations cannot be overridden: allow_matchmaking, challenge_timeout, challenge_filter and block_list.
```
Comment (AttackingOrDefending) [inline config.yml.default:230]: Add a new line at the end of the file.

**Review by AttackingOrDefending — APPROVED** [review]

**Comment by AttackingOrDefending** [comment]
@MarkZH I won't merge this yet, because changes have been done since you approved it. If you approve the new changes, go ahead and merge this.

File: wiki/Configure-lichess-bot.md [hunk 2]
```diff
@@ -237,14 +237,19 @@ will precede the `go` command to start thinking with `sd 5`. The other `go_comma
...
-
-The `challenge_filter` option can be useful if your matchmaking settings result in a lot of declined challenges. The bots that accept challenges will be challenged more often than those that have declined. The filter will remain until lichess-bot quits or the connection with lichess.org is reset.
+    The `challenge_filter` option can be useful if your matchmaking settings result in a lot of declined challenges. The bots that accept challenges will be challenged more often than those that have declined. The filter will remain until lichess-bot quits or the connection with lichess.org is reset.
+  - `block_list`: An indented list of usernames of bots that will not be challenged. If this option is not present, then the list is considered empty.
+  - `overrides`: Create variations on the matchmaking settings above for more specific circumstances. If there are any subsections under `overrides`, the settings below that will override the settings in the matchmaking section. Any settings that do not appear will be taken from the settings above. <br/>
```
Comment (MarkZH) [inline wiki/Configure-lichess-bot.md:243]: Replace the `<br/>` with an empty line below this one.

File: wiki/Configure-lichess-bot.md [hunk 3]
```diff
@@ -237,14 +237,19 @@ will precede the `go` command to start thinking with `sd 5`. The other `go_comma
...
+  The overrides section must have the following:
+    - Name: A unique name must be given for each override. In the example configuration below, `easy_chess960` and `no_pressure_correspondence` are arbitrary strings to name the subsections and they are unique.
+    - List of options: A list of options to override. Only the options mentioned will change when making the challenge. The rest will follow the default matchmaking options. In the example settings below, the blank settings for `challenge_initial_time` and `challenge_increment` under `no_pressure_correspondence` have the effect of deleting these settings, meaning that only correspondence games are possible.
+    
+    For each matchmaking challenge, the default settings and each override have equal probability of being chosen to create the challenge. For example, in the example configuration below, the default settings, `easy_chess960`, and `no_pressure_correspondence` all have a 1/3 chance of being used to create the next challenge. <br/>
```
Comment (MarkZH) [inline wiki/Configure-lichess-bot.md:248]: Replace the `<br/>` with an empty line below this one.

Comment (TheYoBots) [reply wiki/Configure-lichess-bot.md:248]: Override section using newline:
image
Override section using `<br/>`:
image
This is why I used `<br/>`.

Comment (MarkZH) [reply wiki/Configure-lichess-bot.md:248]: I see. These should still be paragraph breaks with extra space, so how about adding a second `<br/>` on the same line in both places?

**Review by MarkZH — APPROVED** [review]

**Pull request merged**